script/openqa-clone-job

```diff
@@ -112,7 +112,12 @@ $OPENQA_SHAREDIR/factory).
 
 =item B<--skip-checks>
 
-Skips additional checks like maintenance update availability.
+Deptrecated option. Logic was inverted so currently check skipped by default. So if you
```

Comment:
@okurz I'm not sure if we have a ticket or something that keep track of this deprecation.

Reply:
We [do now](https://progress.opensuse.org/issues/182834).